src/app/modules/shared_models/user_model.nim

```diff
@@ -124,7 +124,7 @@ QtObject:
     self.endRemoveRows()
     self.countChanged()
 
-  proc isContactWithIdAdded*(self: Model, id: string): bool = 
+  proc isContactWithIdAdded*(self: Model, id: string): bool {.slot.} = 
```

Comment:
@borismelnik exposing proc directly from the model, as it's done here, is against the rules we're trying to establish. Please revert this and add appropriate method to the `view` of the `module` where you need this info, and get what you need that way.